### DISCUSSION TITLE
For large output binaries, any way to get visual feedback that it's being processed?

### DISCUSSION THREAD

**Discussion by vineetparikh** [bloomberg/memray#457] [Q&A]
Hi there!

I'm using memray to profile a video ML script that I'm suspecting has a memory leak somewhere. The output binary is fairly large (~67.7GB) and when I generate a flamegraph the command just hangs without visual feedback. I'm expecting this is because just loading the file is taking a bit, but I'm wondering if there's any way to visually show progress in loading.

Thanks,
Vineet

**Comment by godlygeek** [comment]
That's an absolutely gigantic capture file. There's definitely a few places where we don't have progress bars and should introduce some, but I expect Memray to take a _long_ time to chew through a file that large. Did it eventually succeed?
